apollo-router/src/plugins/telemetry/otlp.rs

```diff
@@ -118,19 +126,26 @@ pub(crate) enum EndpointDefault {
 #[derive(Debug, Clone, Deserialize, Serialize, Default, JsonSchema)]
 #[serde(deny_unknown_fields)]
 pub(crate) struct HttpExporter {
+    /// Headers to send on report requests
     #[serde(default)]
     pub(crate) headers: HashMap<String, String>,
 }
 
 #[derive(Debug, Clone, Deserialize, Serialize, Default, JsonSchema)]
 #[serde(deny_unknown_fields)]
 pub(crate) struct GrpcExporter {
+    /// The optional domain name for tls config.
+    /// Note that domain name is will be defaulted to match the endpoint is not explicitly set.
     #[serde(default)]
     pub(crate) domain_name: Option<String>,
+    /// The optional ca for tls config
     pub(crate) ca: Option<String>,
+    /// The optional cert for tls config
     pub(crate) cert: Option<String>,
+    /// The optional key for tls config
```

Comment:
```suggestion
    /// The optional private key file for TLS configuration.
```

Same question: Is this a path to the file or the contents of the file?

Reply:
It's the content. If users want to insert the contents of a file then they can use the expansion feature of the config.